src/utils/heroku.js

```diff
@@ -30,6 +30,24 @@ export async function signInToHeroku(email, password) {
       return false;
     }
 
-    log.error('failed to sign in to heroku', err);
+    log.error('Failed to sign in to heroku', err);
+  }
+}
+
+export async function createApp(appName, apiKey) {
+  log.verbose('creating heroku app', appName);
+  log.debug('creating heroku app', apiKey);
```

Comment:
You sure we want to log the API key here?

Reply:
removed